tsconfig.json

```diff
@@ -13,9 +13,17 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
+    "incremental": true,
     "jsx": "preserve",
-    "types": ["cypress"]
+    "types": ["@types/testing-library__jest-dom"]
```

Comment:
Are the cypress types still needed? I'm not too familiar with how the typeloading works for multiple testing libraries.

Reply:
Unfortunately yes, they are -- There's another tsconfig which now lives in `/cypress` as well.

Reply:
Oh that's fine, I just missed that TSConfig